src/main/java/general/ui/ChatWindow.java

```diff
@@ -0,0 +1,80 @@
+package general.ui;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * A {@code ChatWindow} formats messages as chat bubbles. There are two
+ * types of messages: commands, which are sent by the user and appear on the left,
+ * and responses, which are sent by the program and appear on the right.
+ *
+ * @author Richard Dominick
+ */
+public class ChatWindow {
+    private final int windowWidth;
+    private final Scanner sc;
+
+    public ChatWindow(int windowWidth) {
+        this.windowWidth = windowWidth;
+        sc = new Scanner(System.in);
+    }
+
+    private static Text[] breakLines(Text data) {
```

Comment:
Is it better to name it getBrokenLines()?

Reply:
Hmm, could you point me to which part specifically? This isn't a getter method by any means, as it takes in a `Text` and returns a broken-up version of it. It does not depend on any internal object state at all.